build/buildRunTimeImageBases.sh

```diff
@@ -81,7 +81,7 @@ dockerFileName="base.$runtimeImageDebianFlavor.Dockerfile"
 stagingDockerFileName="base.$runtimeImageDebianFlavor.staging.Dockerfile"
 dockerFiles=$(find $runtimeImagesSourceDir -type f \( -name $dockerFileName -o -name $stagingDockerFileName \))
 
-busterNodeDockerFiles=()
+nodeDockerfiles=()
```

Comment:
I'm guessing this was re-named because we support bookworm, buster and bullseye for node now but I'm curious why it was named `busterNodeDockerFiles` before? Didn't we support bullseye images from before?

Reply:
When we first introduced support for buster a few years ago, we added this section to build Node base images for _only_ buster, and there was an if-statement that would check the flavor for buster and would add the Dockerfiles to the `dockerfiles` variable. Over the years, this logic has changed a little and the aforementioned if-statement was removed so we could build Node base images for all supported Debian flavors (since we have Node 18 on bullseye and Node 14 and 16 on buster). When we removed the if-statement, we didn't change the name of the variable reflect the universal Debian flavor image building we were doing.